client/app/bundles/shared/components/Textarea.scss

```diff
@@ -0,0 +1,53 @@
+@import "base.scss";
+
+$focus-background-color-darken: rgba(0, 0, 0, 0.4);
+$focus-background-color-lighten: lighten($container-color, 3%);
+$light-font-weight: 300;
+
+.label {
+  color: $container-color;
+  font-family: $font-family;
+  font-size: $small-font;
+  padding: 12px 4px;
+  letter-spacing: 0.0944444em;
+  text-transform: uppercase;
+  transition: 0.25s;
+  transition-timing-function: linear;
+
+  &.active {
+    color: $focus-background-color-lighten;
+  }
+}
+
+.textarea {
+  color: rgba(255, 255, 255, 0.8);
+  box-sizing: border-box;
+  background-color: $container-color;
+  font-family: $font-family;
+  font-size: $font18;
+  line-height: $font30;
+  text-indent: 12px;
```

Comment:
Let's make sure we avoid using text-indent here as well, since it makes the text look funky on multiple lines in the textarea! Sorry I didn't catch this the first time!

Reply:
Not at all! I thought it was intentional, so I left it there. Changes made :)